pymodbus/repl/client/main.py

```diff
@@ -88,12 +88,6 @@ def convert(self, value, param, ctx):
         if value in self.choices:
             return self.typ(value)
 
-        if ctx is not None and ctx.token_normalize_func is not None:
-            value = ctx.token_normalize_func(value)
-            for choice in self.casted_choices:  # pylint: disable=no-member
```

Comment:
I searched for `casted_choices` in git history as well as Click project, and could not find it anywhere.